### ISSUE TITLE
Add history (like bash history) functionnality inside Hammr

### ISSUE THREAD

**Issue by jbremond** [usharesoft/hammr#62]
History is working on Hammr for now, but it not saved into a .hammr_history file. So the next you start hammr, the history is lost

**Comment by jbremond** [comment]
Yes I think you are right @segalaj. The main development should be done in ussclicore project, but the name of the history file should be defined in Hammr. So I keep the issue opened

**Comment by bastienmartin** [comment]
7920
